types/component/AnonymousChat/BaseChatItem.types.ts

```diff
@@ -24,7 +25,7 @@ export interface BaseChatItemProps {
 
 export interface ChatItemMyTextProps extends BaseChatItemProps {
   status?: ChatStatus;
-  chatType?: string;
+  chatType: Anonimity;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Changing `chatType` to be of type `Anonimity` instead of `string` is a good practice. It provides better type safety and makes the code more readable and maintainable.
```diff
-  chatType?: string;
+  chatType: Anonimity;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->